src/main/java/world/anhgelus/molehunt/Game.java

```diff
@@ -127,9 +132,9 @@ public void run() {
                 if (gameWonByMoles()) {
                     winner = new TitleS2CPacket(Text.of("The Moles!"));
                 } else {
-                    winner = new TitleS2CPacket(Text.of("Not the Mole!"));
+                    winner = new TitleS2CPacket(Text.of("The survivors!"));
                 }
-                pm.sendToAll(new SubtitleS2CPacket(Text.of("Moles were " + getMolesAsString())));
+                pm.sendToAll(new SubtitleS2CPacket(Text.of("The moles were " + getMolesAsString())));

```

Comment:
`The moles` should be `The Moles` according to other names

Reply:
I will fix this in another PR (and I don't want to fix it there because it will cause merge issues)

Reply:
There is already merging issue

Reply:
Don't do duplicate PR, it's useless and a bad practice